atlp-rwanda/tech-strikers-bn-frontend
# #175636787 Implementing sign up for Barefoot Nomad

#### What does this PR do?
It allows the user to sign up in our barefoot nomad and register himself
#### Description of Task to be completed?
- Made a signup form
- Made validation on the client-side
- made a post route
- allows the user to be validated
#### How should this be manually tested?
run the following command: 
- open visual studio
- ` git clone https://github.com/atlp-rwanda/tech-strikers-bn-frontend.git `
- `cd tech-strikers-bn-frontend`
- `git checkout ch-signup-175636787`
- `npm install`
- `npm run dev`
You will see a homepage and click signup link in top right of the page 
fill the form and check the email see a confirmation link click it or copy it in the browser You account will be verified
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#175636787](https://www.pivotaltracker.com/story/show/175636787)
#### Screenshots (if appropriate)
![Screenshot from 2021-03-15 15-48-44](https://user-images.githubusercontent.com/57761095/111166945-4d6eb400-85a9-11eb-8fe5-f9075aeebd03.png)
![Screenshot from 2021-03-15 15-49-16](https://user-images.githubusercontent.com/57761095/111166968-5069a480-85a9-11eb-93f9-6299eb431871.png)
![Screenshot from 2021-03-15 15-52-46](https://user-images.githubusercontent.com/57761095/111166972-52cbfe80-85a9-11eb-9c01-981b9892d3b3.png)
![Screenshot from 2021-03-15 15-53-04](https://user-images.githubusercontent.com/57761095/111166979-5495c200-85a9-11eb-8d34-930c31118ca6.png)
![Screenshot from 2021-03-15 15-53-21](https://user-images.githubusercontent.com/57761095/111166983-55c6ef00-85a9-11eb-9f5d-7f4eba461e27.png)
![Screenshot from 2021-03-15 15-54-46](https://user-images.githubusercontent.com/57761095/111166988-56f81c00-85a9-11eb-951f-75ec34163d8f.png)

#### Questions:
